### ISSUE TITLE
omit tar warnings

### ISSUE THREAD

**Issue by gilligan** [svanderburg/npm2nix#6]
During the build tar warns about unknown tar header files. Even for medium sized projects this gets extremely noisy:
```
tar: Ignoring unknown extended header keyword 'NODETAR.follow'
tar: Ignoring unknown extended header keyword 'NODETAR.ignoreFiles.0'
tar: Ignoring unknown extended header keyword 'NODETAR.ignoreFiles.1'
tar: Ignoring unknown extended header keyword 'NODETAR.ignoreFiles.2'
tar: Ignoring unknown extended header keyword 'NODETAR.package.name'
tar: Ignoring unknown extended header keyword 'NODETAR.package.version'
tar: Ignoring unknown extended header keyword 'NODETAR.package.description'
tar: Ignoring unknown extended header keyword 'NODETAR.package.license'
tar: Ignoring unknown extended header keyword 'NODETAR.package.repository'
tar: Ignoring unknown extended header keyword 'NODETAR.package.author.name'
tar: Ignoring unknown extended header keyword 'NODETAR.package.author.email'
tar: Ignoring unknown extended header keyword 'NODETAR.package.engines.node'
tar: Ignoring unknown extended header keyword 'NODETAR.package.scripts.test'
tar: Ignoring unknown extended header keyword 'NODETAR.package.files.0'
tar: Ignoring unknown extended header keyword 'NODETAR.package.files.1'
tar: Ignoring unknown extended header keyword 'NODETAR.package.files.2'
tar: Ignoring unknown extended header keyword 'NODETAR.package.keywords.0'
tar: Ignoring unknown extended header keyword 'NODETAR.package.keywords.1'
tar: Ignoring unknown extended header keyword 'NODETAR.package.keywords.2'
tar: Ignoring unknown extended header keyword 'NODETAR.package.keywords.3'
tar: Ignoring unknown extended header keyword 'NODETAR.package.keywords.4'
tar: Ignoring unknown extended header keyword 'NODETAR.package.keywords.5'
tar: Ignoring unknown extended header keyword 'NODETAR.package.devDependencies.chai'
tar: Ignoring unknown extended header keyword 'NODETAR.package.devDependencies.mocha'
```

It should be possible to omit those using `--warning=no-unknown-keyword`.

**Comment by svanderburg** [comment]
Implemented a workaround for this in revision 5100226

**Comment by gilligan** [comment]
Seems to work. Thanks!
